config/config.exs

```diff
@@ -8,15 +8,17 @@
 use Mix.Config
 
 config :safira,
-  ecto_repos: [Safira.Repo]
+  ecto_repos: [Safira.Repo],
+  company_code: System.get_env("COMPANY_CODE"),
+  speaker_code: System.get_env("SPEAKER_CODE"),
+  staff_code: System.get_env("STAFF_CODE")
 
 # Configures the endpoint
 config :safira, SafiraWeb.Endpoint,
   url: [host: "localhost"],
   secret_key_base: "3KpMz5Dsmzm2+40c8Urp8UC0N95fFWvsHudtIUHjTv2yGsikjN3wIHPNPi3e+4xi",
   render_errors: [view: SafiraWeb.ErrorView, accepts: ~w(json)],
```

Comment:
wait, I know it wasn't changed in this PR, but isn't there any problem with a secret being exposed?

Reply:
There is no problem since this secret is only used in dev and test configurations. The prod configuration overwrites this value.